### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by smedipally**
Python language for Code Review project

### CODE REVIEW

File: F21/smedipally/.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+            letter=word.lower()
+
+        print(letter)
+
+   
+
+    if(lang=='english' or lang=='en' or lang=='English' or lang=='en-US' or lang=='en-IE' or lang=='en-Latn'):
+
+        for letter in word:
```
Comment (hpakala1) [inline F21/smedipally/.py:83]: using for loops is best strategy that reduced your code line.

File: F21/smedipally/.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
+def LowerCaseConverter(word,lang):
+
```
Comment (hpakala1) [inline F21/smedipally/.py:2]: Can we follow the universal style guide for the line alignment

File: F21/smedipally/.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    lang=lang
```
Comment (hpakala1) [inline F21/smedipally/.py:3]: The parametres can see the variables inside the function without the assignment of these variables

File: F21/smedipally/.py (hunk 1 above)
Comment (smedipally) [reply F21/smedipally/.py:83]: Thank you

File: F21/smedipally/.py (hunk 2 above)
Comment (smedipally) [reply F21/smedipally/.py:2]: Yes, We can! But for better understanding of code i did it on purpose!

File: F21/smedipally/.py (hunk 3 above)
Comment (smedipally) [reply F21/smedipally/.py:3]: Okay! I will check this out!

File: F21/smedipally/.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+    word=word
```
[the remark this answers is not in the document]
Comment (smedipally) [reply F21/smedipally/.py:5]: Sure! I will check this out too!

**Pull request closed without merging**